ISISComputingGroup/ibex_gui
# Ticket913 better name for synoptic component

To test:
- Check a selection of old synoptics work ok
- Check selecting Goniometer or DAE in the component type that compomnent target details can not be set any more
- Check switching between Goniometer/DAE and another component type that the previous selected target is not lost
- Check new components, both Goniometer/DAE and opi based work correctly in the synoptic
